src/core/Akka.API.Tests/CoreAPISpec.ApproveCluster.approved.txt

```diff
@@ -59,7 +59,7 @@ namespace Akka.Cluster
     {
         public ClusterActorRefProvider(string systemName, Akka.Actor.Settings settings, Akka.Event.EventStream eventStream) { }
         protected override Akka.Actor.IActorRef CreateRemoteWatcher(Akka.Actor.Internal.ActorSystemImpl system) { }
-        public override void Init(Akka.Actor.Internal.ActorSystemImpl system) { }
```

Comment:
Can we move the `Akka.Cluster` changes into a separate PR?

Reply:
yes, its in an other PR https://github.com/akkadotnet/akka.net/pull/5445, 
made a mistake with the branch, will try to resolve it